src/cablab/providers/air_temperature.py

```diff
@@ -29,7 +27,7 @@ def get_variable_descriptors(self):
         return {
             VAR_NAME: {
                 'data_type': numpy.float32,
-                'fill_value': -32767,
+                'fill_value': FILL_VALUE,
                 'units': 'K',
                 'long_name': '2 metre temperature',
                 'scale_factor': 0.0019718202938428923,
```

Comment:
Yes, I think this would be ideal. I don't see a reason to keep the offset and scale_factor, because we convert this to Float32 anyway. I think it would be nice to have an offset-free cube, but it would be ok, too if there are strong other opinions...